examples/vlc_server_driver_example.py

```diff
@@ -38,37 +92,55 @@ def plot_lc(lc, image_to_pose):
 
 
 plugins = ob.discover_plugins()
+print("Discovered Plugins: ")
 print(plugins)
 
 data_path = "/home/aaron/lxc_datashare/uHumans2_apartment_s1_00h.bag"
 rgb_topic = "/tesse/left_cam/rgb/image_raw"
 rgb_info_topic = "/tesse/left_cam/camera_info"
+depth_topic = "/tesse/depth_cam/mono/image_raw"
+depth_info_topic = "/tesse/depth_cam/camera_info"
 body_frame = "base_link_gt"
 body_frame = "base_link_gt"
 map_frame = "world"
 
 loader = RosbagDataLoader(
-    data_path, rgb_topic, rgb_info_topic, body_frame=body_frame, map_frame=map_frame
+    data_path,
+    rgb_topic,
+    rgb_info_topic,
+    body_frame=body_frame,
+    map_frame=map_frame,
+    depth_topic=depth_topic,
+    depth_info_topic=depth_info_topic,
```

Comment:
Not sure that the dataloader takes in the depth info (unless I forgot that I implemented rectification)